polls/templates/polls/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
```

Comment:
Same here. You can create a base.html file and extend from that so you dont have to redeclare the doctype, meta tags, <head>, etc. and just "inject" code into the base template.